_blog.yml

```
@@ -475,3 +475,11 @@
   date: January 25, 2022
   tags:
   - guide
+- local: asr-chunking
+  title: "Making automatic speech recognition work on large files with Wav2Vec2 in 🤗 Transformers"
```

I would go with ASR as that's used in other titles (and to make it shorter)
```suggestion
  title: "Making ASR work on large files with Wav2Vec2 in 🤗 Transformers"
```

---

I really think title should be spelled out. At the very least for SEO and also not to exclude non connoisseurs.

---

Sounds good! I was going with consistency but I agree with you